sycl/unittests/Extensions/CommandGraph.cpp

```diff
@@ -1952,13 +1952,15 @@ TEST_F(MultiThreadGraphTest, BeginEndRecording) {
       Queue.get_context(), Queue.get_device()};
 
   for (unsigned i = 0; i < NumThreads; ++i) {
-    queue MyQueue{Queue.get_context(), Queue.get_device()};
+    queue MyQueue;
```

Comment:
```suggestion
    queue MyQueue{Queue.get_context(), Queue.get_device()};
```
We should still enforce a matching context and device, no?

Reply:
Yes. You are right :+1: